Library/Formula/mdbtools.rb

```diff
@@ -17,7 +17,7 @@ class Mdbtools < Formula
   def install
     ENV.deparallelize
 
-    args = ["--prefix=#{prefix}"]
+    args = ["--prefix=#{prefix}","--with-unixodbc=/usr/local"]
```

Comment:
Please add a space after the comma. If this `/usr/local` refers to Homebrew’s prefix please use `"--with-unixodbc=#{HOMEBREW_PREFIX}"` instead.